src/components/shared/forms/ProjectDetailsForm.tsx

```diff
@@ -75,10 +74,7 @@ export default function ProjectDetailsForm({
             name="logoUri"
             initialUrl={form.getFieldValue('logoUri')}
             onSuccess={logoUri => {
-              const prevUrl = form.getFieldValue('logoUri')
-              // Unpin previous file
               form.setFieldsValue({ logoUri })
-              if (prevUrl) unpinIpfsFileByCid(cidFromUrl(prevUrl))
```

Comment:
This will mean files will pile up, but this is a low-volume interaction, so not too bad.

We can solve once we move to Next.